docs/ccv-distribution/ccv-distribution.md

```diff
@@ -1,92 +0,0 @@
-# CCV Distribution
```

Comment:
This document is useful. Can we move it do a issue on how to improve the distribution. We may need this for other iterations of ICS.

Reply:
Yea I can add this back, is it still representative of the code how it exists now?

Reply:
I think this file should still be removed as it's not documentation on how the current code in main works. The diagram is maintained and could be addressed in https://github.com/cosmos/interchain-security/issues/760